README.md

```diff
@@ -62,3 +62,20 @@ In development:
 composer drush audit:best-practices
 ```
 This will run the given Drush command against a local test Drupal site, the "system under test". Run the tests once to install the Drupal site.
+
+
+Testing via Tagged Deploy
+-------------------------
+
+* After your work is ready to test in a production environment, cut a new RC tag from your branch and push that tag to github.
```

Comment:
Does there need to be an RC tag, or can you just deploy from the dev branch? The instructions that follow look like they just deploy from whatever dev files exist on the local system.

Reply:
It won't use the local files, the command in question that uses the variable is `/build/composer create-project pantheon-systems/site-audit-tool:$site_audit_tool_version "/build/extensions/d8/Commands/site-audit-tool" --no-dev --ignore-platform-reqs`, which I _believe_ only works with tags, though it may work with the remote branch.